pulseapi/entries/models.py

```diff
@@ -27,7 +46,18 @@ def public(self):
         """
         Return all entries to start
         """
-        return self.filter(is_approved=True)
+        try:
+            # This has to happen in a try/catch, so that migrations
```

Comment:
no, it's needed for the migration *path*, so even in dev setting it stays necessary =(

downside of django.